Okta.AspNet/Okta.AspNet.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Official Okta middleware for ASP.NET 4.6.2+. Easily add authentication and authorization to ASP.NET applications.</Description>
     <Copyright>(c) 2019 Okta, Inc.</Copyright>
-    <Version>3.2.2</Version>
+    <Version>3.2.3</Version>
     <Authors>Okta, Inc.</Authors>
     <TargetFramework>net462</TargetFramework>
```

Comment:
Was this 4.6.2 already? when did we change this?

Reply:
It was.  Good point 😅.

Reply:
Interesting. We still need to release a patch which is basically a new build with the updated abstractions dependency.